gatewayconfig/gatewayconfig_app.py

```diff
@@ -75,14 +79,22 @@ def stop(self):
         # Quits the cputemp application
         self.bluetooth_services_processor.quit()
 
-    def init_sentry(self, sentry_dsn, balena_app_name, balena_device_uuid, variant, firmware_version):
+    def init_sentry(self, sentry_dsn, release, balena_id, balena_app):
+        """
+        Initialize sentry with balena_id and balena_app as tag.
+        If sentry_dsn is not set, do nothing.
+        """
+
+        if not sentry_dsn:
+            return
+
         sentry_sdk.init(
             sentry_dsn,
-            environment=balena_app_name,
-            release=f"hm-config@{firmware_version}"
+            release=f"hm-config@{release}"
         )
-        sentry_sdk.set_user({"id": balena_device_uuid})
-        sentry_sdk.set_context("variant", {variant})
+
+        sentry_sdk.set_tag("balena_id", balena_id)
+        sentry_sdk.set_tag("balena_app", balena_app)
```

Comment:
why are we removing variant. I don't know why it was added but we have to assume that it was added on purpose.

Reply:
`balena_app` is the fleet name and it contains the variant. I don't think we need to keep `variant` too.

Reply:
we shouldn't depend on that, that could change over time.

Reply:
Ok, make sense. It's reverted now.